code-challenges.test.js

```diff
@@ -24,15 +24,45 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
-
+var fibonacciInputOne = 6
+var fibonacciInputTwo = 10
+
+describe("when 'fibonacciArray' is called", () => {
+  let fibonacciOutputOne = [1, 1, 2, 3, 5, 8]
+  let fibonacciOutputTwo = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+  it("returns an array with the amount of numbers requested in the Fibonacci sequence", () => {
+    expect(fibonacciArray(fibonacciInputOne)).toEqual(fibonacciOutputOne)
+    expect(fibonacciArray(fibonacciInputTwo)).toEqual(fibonacciOutputTwo)
+  })
+})
 
 // b) Create the function that makes the test pass.
-
-
-
-
+//make a function that takes a single number as an arguement (greater than 2) to determine length of array
+//create new array
+//map will then return value + value to each iteration
+
+// const fibonacciArray = (i) => {
+//   let newFibArray = []
+//   return newFibArray.forEach((item, i) => {
+//     (item + item++)
+//   });
+//   return newFibArray
+// }
+
+//^could not use map or foreach -- while I am sure there is a way....could not get it to work.  Decided to go for a for loop instead
+//used the for loop to push into array.  Since we will have an input greater than 2, the array will start with the first two digits
+//for loop will start on index 2 -- run for the length of user input number - and iterate until complete
+//iteration will push an equation that takes the 2 previous index and adds them together to become the sum on current index
+
+const fibonacciArray = (length) => {
```

Comment:
this works too! What if the function is called with 0 or 1? It seems this would return `[1,1]`